### ISSUE TITLE
Unable to build due to unresolved import from simdeez

### ISSUE THREAD

**Issue by dominictwlee** [jackmott/rust-simd-noise#18]
simdnoise version: 3.1.4

```
error[E0432]: unresolved import `simdeez::overloads`
  --> /home/dominiclee/.cargo/registry/src/github.com-1ecc6299db9ec823/simdnoise-3.1.4/src/avx2.rs:13:14
   |
13 | use simdeez::overloads::*;
   |              ^^^^^^^^^ could not find `overloads` in `simdeez`

error[E0432]: unresolved import `simdeez::overloads`
 --> /home/dominiclee/.cargo/registry/src/github.com-1ecc6299db9ec823/simdnoise-3.1.4/src/scalar.rs:3:14
  |
3 | use simdeez::overloads::*;
  |              ^^^^^^^^^ could not find `overloads` in `simdeez`

error[E0432]: unresolved import `simdeez::overloads`
  --> /home/dominiclee/.cargo/registry/src/github.com-1ecc6299db9ec823/simdnoise-3.1.4/src/sse2.rs:10:14
   |
10 | use simdeez::overloads::*;
   |              ^^^^^^^^^ could not find `overloads` in `simdeez`

error[E0432]: unresolved import `simdeez::overloads`
  --> /home/dominiclee/.cargo/registry/src/github.com-1ecc6299db9ec823/simdnoise-3.1.4/src/sse41.rs:10:14
   |
10 | use simdeez::overloads::*;
   |              ^^^^^^^^^ could not find `overloads` in `simdeez`

error: aborting due to 4 previous errors

For more information about this error, try `rustc --explain E0432`.
error: could not compile `simdnoise`.

Caused by:
  process didn't exit successfully: `rustc --crate-name simdnoise --edition=2018 /home/dominiclee/.cargo/registry/src/github.com-1ecc6299db9ec823/simdnoise-3.1.4/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type lib --emit=dep-info,metadata,link -C debuginfo=2 -C metadata=5a0c5a6844d71316 -C extra-filename=-5a0c5a6844d71316 --out-dir /home/dominiclee/Projects/simdnoise-tester/target/debug/deps -L dependency=/home/dominiclee/Projects/simdnoise-tester/target/debug/deps --extern simdeez=/home/dominiclee/Projects/simdnoise-tester/target/debug/deps/libsimdeez-91dc463eee65cdef.rmeta --cap-lints allow` (exit code: 1)
```

**Comment by jackmott** [comment]
sorry about that, fix incoming

**Comment by jackmott** [comment]
latest version should fix, let me know, thanks.

**Comment by dominictwlee** [comment]
Yep, fixed in 3.1.5, thanks!
